frontend/src/pages/acceleratorProfiles/screens/manage/ManageAcceleratorProfileFooter.tsx

```diff
@@ -33,34 +31,31 @@ export const ManageAcceleratorProfileFooter: React.FC<ManageAcceleratorProfileFo
   const [isLoading, setIsLoading] = React.useState<boolean>(false);
   const navigate = useNavigate();
   const notification = useNotification();
+  const { dashboardNamespace } = useDashboardNamespace();
 
   const onCreateAcceleratorProfile = async () => {
     setIsLoading(true);
-    createAcceleratorProfile(state)
-      .then((res) => {
-        if (res.success) {
-          if (redirectPath !== '/acceleratorProfiles') {
-            notification.success(
-              'Accelerator profile has been created.',
-              <Stack hasGutter>
-                <StackItem>
-                  A new accelerator profile <strong>{state.displayName}</strong> has been created.
-                </StackItem>
-                <StackItem>
-                  <Button isInline variant="link" onClick={() => navigate(`/acceleratorProfiles`)}>
-                    View profile details
-                  </Button>
-                </StackItem>
-              </Stack>,
-            );
-          }
-          navigate(redirectPath);
-        } else {
-          setErrorMessage(res.error || 'Could not create accelerator profile');
+    createAcceleratorProfile(state, dashboardNamespace)
+      .then(() => {
+        if (redirectPath !== '/acceleratorProfiles') {
+          notification.success(
+            'Accelerator profile has been created.',
+            <Stack hasGutter>
+              <StackItem>
+                A new accelerator profile <strong>{state.displayName}</strong> has been created.
+              </StackItem>
+              <StackItem>
+                <Button isInline variant="link" onClick={() => navigate(`/acceleratorProfiles`)}>
+                  View profile details
+                </Button>
+              </StackItem>
+            </Stack>,
+          );
         }
+        navigate(redirectPath);
       })
       .catch((err) => {
-        setErrorMessage(err.message);
+        setErrorMessage(err.message || 'Could not create accelerator profile');
```

Comment:
There should always be an error message for the k8s call, so we can remove the content after `||`. Same below.